contracts/examples/chainlink-solana-demo/Anchor.toml

```diff
@@ -1,13 +1,16 @@
 [programs.localnet]
-hello_world = "Fg6PaFpoGXkYsidMpWTK6W2BeZ7FEfcYkg476zPFsLnS"
+chainlink_solana_demo = "Fg6PaFpoGXkYsidMpWTK6W2BeZ7FEfcYkg476zPFsLnS"
+
+[programs.devnet]
+chainlink_solana_demo = "JC16qi56dgcLoaTVe4BvnCoDL6FhH5NtahA7jmWZFdqm"
 
 [registry]
 url = "https://anchor.projectserum.com"
 
 [provider]
-cluster = "localnet"
-# wallet = "~/.config/solana/id.json"
-wallet = "../../id.json"
+cluster = "devnet"
+wallet = "./id.json"
```

Comment:
I disagree with this & the change in tests, users should be testing their code locally on localnet. That's why I added the `[test.genesis]` section that includes the store program in the test chain, then users can test against an actual store they're able to submit different prices to. That way different scenarios can be tested.

A similar pattern is used for the anchor `cfo` example: it pulls in `dex` and other contracts under `deps/` https://github.com/project-serum/anchor/tree/master/tests/cfo/deps

Then it has a `utils.js` file that contains the boilerplate of configuring those contracts: https://github.com/project-serum/anchor/blob/master/tests/cfo/tests/utils/index.js